test/addons/worker-buffer-callback/binding.cc

```diff
@@ -30,6 +32,7 @@ void Initialize(Local<Object> exports,
                    data,
                    sizeof(data),
                    [](char* data, void* hint) {
+                     free(data);
```

Comment:
Just curious, why these changes? Is that about the V8 problem when creating backing stores for the same pointer multiple times?

In any case, you can’t mix `new` and `free()`, so this should be `delete data;`.

Reply:
Yes, it's the problem with the new API. We have a 1-1 relationship between BackingStore and buffer address, hence calling NewBackingStore multiple times with the same buffer will throw an error.
Oh right, i was jumping between C and C++, thank you for pointing this out.